python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -297,3 +304,30 @@ def sort_values(
         df4.divisions = tuple(methods.tolist(divisions))
 
     return df4
+
+
+def _set_shuffle(shuffle):
+    # Utility to set the `shuffle`-kwarg default
+    # and validate a user-specified option
```

Comment:
We also side-effectfully monkey-patch `rearrange_by_column_tasks`.

That attempted import only really needs to happen once, so how about:

```python
try:
    import dask_cuda  # noqa: F401
    def _get_shuffle_type(shuffle):
        shuffle = shuffle or dask.config.get("shuffle", "tasks")
        if shuffle not in {"tasks", "explicit-comms"}:
             raise ValueError(...)
        return shuffle
except ImportError:
    def _get_shuffle_type(shuffle):
         shuffle = shuffle or dask.config.get("shuffle", "tasks")
         if shuffle == "explicit-comms":
             raise ValueError(...)
         elif shuffle != "tasks":
             raise ValueError(...)
         return shuffle
```